Source/Scripting/angelscript/add_on/scripthelper/scripthelper.cpp

```diff
@@ -228,7 +228,7 @@ int WriteConfigToStream(asIScriptEngine *engine, ostream &strm)
 			for(;;)
 			{
 				// Find " characters
-				pos = str.find("\"",pos);
+				pos = str.find('\"',pos);
```

Comment:
this code is inside angelscript I think? I believe these local patches may make it harder to diff when upgrading the angelscript version.

Reply:
or maybe this is a local "add-on". not sure if it's one of the standard ones or not.